### ISSUE TITLE
doesnt compile/flash when i click the button...

### ISSUE THREAD

**Issue by Adarsh881** [emmebrusa/TSDZ2-Smart-EBike-1#8]
Hi. 
I did everything what was written in the instruction manual but in the end it doesnt compile/flash when i click the button.

I took a picture off the screen. sorry...printscreen is not working...
Thank you:)

![IMG_20200919_010046 1367](https://user-images.githubusercontent.com/71426027/93652035-e31ef180-fa13-11ea-98b5-9a1df8a64bfb.jpg)

**Comment by eIinx** [comment]
SDCC and STVP must be installed inside the root for the bat-script that is reponsable for compile&flash

**Comment by jay7019** [comment]
Adarsh881 did you figue this out im having same problem

**Comment by Adarsh881** [comment]
Yes if I remember correctly you simply have to run it in windows administrator mode then it worked...

**Comment by jay7019** [comment]
Thanks for your help


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
